schemas/constructs/v1beta1/designs.json

```diff
@@ -1,63 +1,53 @@
-  "type": "object",
-  "properties": {
-    "id": {
-      "$ref": "../core.json#/definitions/uuid"
-    },
-    "name": {
-      "type": "string",
-      "description": "Name of the design; a descriptive, but concise title for the design document."
-    },
-    "schemaVersion": {
-      "$ref": "../core.json#/definitions/versionString",
-      "description": "Specifies the version of the schema to which the design conforms."
-    },
-    "version": {
-      "$ref": "../core.json#/definitions/semverString",
-      "description": "Revision of the design as expressed by an auto-incremented, SemVer-compliant version number. May be manually set by a user or third-party system, but will always be required to be of version number higher than the previously defined version number.",
-      "minLength": 1,
-      "maxLength": 50
-    },
-    "components": {
-      "description": "A list of one or more component declarations.",
-      "minItems": 0,
-      "type": "array",
-      "items": {
-        "$ref": "./component.json"
-      }
-    },
-    "preferences": {
-      "type": "object",
-      "description": "Design-level preferences",
-      "properties": {
-        "layers": {
-          "type": "array",
-          "description": "List of available layers",
-          "items": {
-            "type": "string"
-          }
-        }
-      },
-      "required": [
-        "layers"
-      ]
-    },
-    "relationships": {
-      "description": "List of relationships between components",
-      "type": "array",
-      "items": {
-        "$ref": "../v1alpha3/relationship.json"
-      }
-    }
-  },
-  
-  "required": [
-    "name",
-    "schemaVersion",
-    "version",
-    "components"
-  ]
-}
\ No newline at end of file
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Design Schema",
+  "description": "Designs are your primary tool for collaborative authorship of your infrastructure, workflow, and processes.",
+  "type": "object",
+  "properties": {
+    "id": {
+      "$ref": "../core.json#/definitions/uuid"
+    },
+    "name": {
+      "type": "string",
+      "description": "Name of the design; a descriptive, but concise title for the design document."
+    },
+    "schemaVersion": {
+      "$ref": "../core.json#/definitions/versionString",
+      "description": "Specifies the version of the schema to which the design conforms."
+    },
+    "version": {
+      "$ref": "../core.json#/definitions/semverString",
+      "description": "Revision of the design as expressed by an auto-incremented, SemVer-compliant version number. May be manually set by a user or third-party system, but will always be required to be of version number higher than the previously defined version number.",
+      "minLength": 1,
+      "maxLength": 50
+    },
+    "components": {
+      "description": "A list of one or more component declarations.",
+      "minItems": 0,
+      "type": "array",
+      "items": {
+        "$ref": "./component.json"
```

Comment:
@codeSafari10 do we need to have a version listed here?

Reply:
@MUzairS15, @aabidsofi19, you might confirm.

Reply:
i think we dont need, because this is pointing to the v1beta1 of components

Reply:
The design schema is inside the `v1beta1` and hence `./component.json` points to the `v1beta1` of the components.